src/main/java/sirius/db/jdbc/schema/SchemaUpdateAction.java

```diff
@@ -116,13 +116,13 @@ public String getError() {
     public void execute(Database db) {
         error = null;
         for (String statement : getSql()) {
-            if (TaskContext.get().isActive()) {
-                try {
-                    OMA.LOG.FINE("Executing Schema Update: %s", statement);
-                    db.createQuery(statement).executeUpdate();
-                } catch (Exception e) {
-                    error = e.getMessage();
+            try {
+                if (!Sirius.isDev()) {
+                    OMA.LOG.INFO("Executing Schema Update: %s", statement);
                 }
+                db.createQuery(statement).executeUpdate();
+            } catch (Exception e) {
+                error = e.getMessage();
```

Comment:
Bissle uncool, dass nur der letzte Fehler gespeichert wird. Eigentlich ist doch der erste meistens interessanter?

Reply:
Wäre ne Liste von Errors da evtl. besser? Ist die Message hier immer gesetzt oder muss man sich ggf. in bestimmten Fällen mit getCause() hochhangeln?

Reply:
wäre auch für ne liste von exceptions und eher vom rootCause die message (Exceptions.getRootCause).